job_bundles/README.md

```diff
@@ -0,0 +1,277 @@
+3. A special parameter that is passed to the [deadline:CreateJob API](https://docs.aws.amazon.com/deadline-cloud/latest/APIReference/API_CreateJob.html)
+   when creating a Job with the Job Bundle:
+    * `deadline:priority` - The value must be an integer, and is passed as the `priority` request parameter to the API.
+    * `deadline:targetTaskRunStatus` - Value must be a string, and is passed as the `targetTaskRunStatus` request parameter to the API.
+    * `deadline:maxFailedTasksCount` - Value must be an integer, and is passed as the `maxFailedTasksCount` request parameter to the API.
+    * `deadline:maxRetriesPerTask` - Value must be an integer, and is passed as the `maxRetriesPerTask` request parameter to the API.
+
+While a Job Template is always a template rather than a specific Job to run, the parameter values file allows
+a Job Bundle to act as a template if some Job Parameters do not have values defined in this file, or as a specific job submission
+if all Job Parameters have values.
+
+For example, the [`blender_render` sample](https://github.com/aws-deadline/deadline-cloud-samples/tree/mainline/job_bundles/blender_render)
+has no parameter values file, and its Job Template defines Job Parameters, like `BlenderSceneFile` and `OutputDir`, that
+have no default values, so it must be used as a template from which Jobs are created. When we create a Job using this Job Bundle, with
+`deadline bundle gui-submit blender_render/`, a new Job Bundle is written to the job history directory and that Job Bundle has
+a `parameter_values.yaml` file that contains the values of all parameters defined for the Job submission:
+
+```bash
+% cat ~/.deadline/job_history/\(default\)/2024-06/2024-06-20-01-JobBundle-Demo/parameter_values.yaml
+parameterValues:
+- name: deadline:targetTaskRunStatus
+  value: READY
+- name: deadline:maxFailedTasksCount
+  value: 10
+- name: deadline:maxRetriesPerTask
+  value: 5
+- name: deadline:priority
+  value: 75
+- name: BlenderSceneFile
+  value: /private/tmp/bundle_demo/bmw27_cpu.blend
+- name: Frames
+  value: 1-10
+- name: OutputDir
+  value: /private/tmp/bundle_demo/output
+- name: OutputPattern
+  value: output_####
+- name: Format
+  value: PNG
+- name: CondaPackages
+  value: blender
+- name: RezPackages
+  value: blender
+```
+
+We could recreate the exact same Job by submitting this new Job Bundle instead with: 
+
+```
+deadline bundle submit ~/.deadline/job_history/\(default\)/2024-06/2024-06-20-01-JobBundle-Demo/
+```
+
+Note: The submitted Job Bundle is saved to your "job history directory" and the location of that directory 
+can be found by running `deadline config get settings.job_history_dir`.
+
+### Elements - Asset References
+
+The `asset_references.json`/`asset_references.yaml` file in a Job Bundle is an interface for Deadline Cloud's
+Job Attachments feature that lists input files and directories, as well as output directories for the Jobs that you
```

Comment:
Can we add a link for [Job Attachments](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/storage-job-attachments.html#:~:text=Job%20attachments%20enable%20you%20to,bucket%20for%20your%20job%20attachments.)